delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/bio/BioAssertions.scala

```diff
@@ -43,7 +43,7 @@ trait BioAssertions { self: Assertions =>
       )
     }
 
-    def assert(expected: A, clue: Any = "values are not the same"): UIO[Unit] = io.redeemCause(
+    def assert(expected: A, clue: Any = "values are not the same")(implicit loc: Location): UIO[Unit] = io.redeemCause(
```

Comment:
This might already be coming from the class it is in

Reply:
This just changes the position of the error to be where this assert method is called

Reply:
(You can never not have a location, I believe. But when writing assertion helpers like this, adding this location makes navigation to the "real" error easier)